.github/workflows/automated-tests.yml

```
@@ -578,29 +578,28 @@ jobs:
         with:
-          name: blob-report-*
-          failOnError: false
+          node-version: 22
+      - name: Check if blob reports exist
+        if: ${{ github.event.workflow_run.conclusion != 'success' && fromJSON(env.hasFailure) }}
+        id: check-blob-reports
+        uses: ./.github/actions/check-artifact-exists
+        with:
+          artifact-name: blob-report-
+      - name: Merge and upload the blob reports
+        if: ${{ fromJSON(env.hasFailure) && fromJSON(steps.check-blob-reports.outputs.result || 'false') }}
```

actionlint (1.7.10)

property "result" is not defined in object type {exists: string}

(expression)